documentation/pe_puppetdb_notes.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: "PuppetDB 3.2 » PE PuppetDB Notes"
+layout: default
+canonical: "/pe-puppetdb-notes"
+---
+
+[reports]: ./api/query/v4/reports.html
+[terminus]: ./puppetdb_connection.markdown#include_unchanged_resources
+
+## 2015.3.0
```

Comment:
I'm not really opinionated here, we just need somewhere to tell users about unchanged resources as a feature and how to turn it off, to address the concern of CS folks. Wherever that lives for now is cool. Maybe I'll just remove this file and leave the other 2 changes I have, then we can make a different ticket to address the PE docs problem in general?